### PR TITLE
Fix #532, to consider an extension of PBO

### LINKED ISSUE #532: Consider an extension of Parameter Block OBU in terms of a new parameter definition type

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/537.html" title="Last updated on Jun 27, 2023, 10:20 PM UTC (3228ec6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/537/3318721...3228ec6.html" title="Last updated on Jun 27, 2023, 10:20 PM UTC (3228ec6)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1409,6 +1413,10 @@ class parameter_block_obu() {
 
 Each value of duration, constant_subblock_duration and subblock_duration SHALL be expressed as the number of ticks at the [=parameter_rate=] specified in the corresponding parameter definition.
 
+<dfn noexport>parameter_data_size</dfn> indicates the size in bytes of a new parameter data with [=param_definition_type=] = p (where, p > 2) immediately following this field.
```
Comment (sunghee-hwang) [inline index.bs:1416]:
```suggestion
<dfn noexport>parameter_data_size</dfn> indicates the size in bytes of [=parameter_data_bytes=].
```

File: index.bs [hunk 2]
```diff
@@ -1409,6 +1413,10 @@ class parameter_block_obu() {
...
+
+<dfn noexport>parameter_data_bytes</dfn> indicates the byte representations of the syntaxes of the new parameter data with [=param_definition_type=] = p (where, p > 2).
```
Comment (sunghee-hwang) [inline index.bs:1418]:
```suggestion
<dfn noexport>parameter_data_bytes</dfn> represents reserved bytes for future use when new syntaxes are defined. Parsers compliant to this version of the specification SHALL ignore these bytes.
```

**Review by sunghee-hwang — COMMENTED** [review]
To address the comment

**Review by cconcolato — APPROVED** [review]

**Review by tdaede — APPROVED** [review]

File: index.bs [hunk 3]
```diff
@@ -1387,6 +1387,10 @@ class parameter_block_obu() {
     if (param_definition_type == PARAMETER_DEFINITION_RECON_GAIN) {
       recon_gain_info_parameter_data();
     }
+    if (param_definition_type > 2) {
```
Comment (tdaede) [inline index.bs:1390]: this can be an } else {

File: index.bs [hunk 4]
```diff
@@ -1381,12 +1381,16 @@ class parameter_block_obu() {
     if (param_definition_type == PARAMETER_DEFINITION_MIX_GAIN) {
       mix_gain_parameter_data();
     }
-    if (param_definition_type == PARAMETER_DEFINITION_DEMIXING) {
+    els if (param_definition_type == PARAMETER_DEFINITION_DEMIXING) {
```
Comment (sunghee-hwang) [inline index.bs:1384]:
```suggestion
    else if (param_definition_type == PARAMETER_DEFINITION_DEMIXING) {
```

**Pull request merged**